compound_functions.js

```diff
@@ -0,0 +1,192 @@
+// var {graphql, buildSchema} = require('graphql');
+
```

Comment:
ok maybe cleanup the aave , compound functions files a little bit to avoid redundancies.